test/cmd/builds.sh

```diff
@@ -19,6 +19,18 @@ cat "${OS_ROOT}/Dockerfile" | oc new-build -D - --name=test
 oc get bc/test
 oc new-build --dockerfile=$'FROM centos:7\nRUN yum install -y httpd'
 oc get bc/centos
+oc new-build -D $'FROM openshift/origin:v1.1' --to-docker
+[[ $(oc get bc/origin --template '{{with .spec.output.to}}{{.kind}} {{.name}}{{end}}') == "DockerImage library/origin:latest" ]]
```

Comment:
@smarterclayton I would like your input here.

Currently, `oc new-build -D $'FROM openshift/origin:v1.1' --to-docker` configures the build output to a DockerImage "origin:latest".

Should we keep it that way? Or perhaps require --to to be specified.